src/query/functions/tests/it/scalars/map.rs

```diff
@@ -182,6 +183,11 @@ fn test_map_keys(file: &mut impl Write) {
     );
 }
 
+fn test_map_contains_key(file: &mut impl Write) {
+    run_ast(file, "map_contains_key({'k1': 'v1', 'k2': 'v2'}, 'k1')", &[
```

Comment:
add some test case with empty map and other key types.

Reply:
By the way, please add a few sql logic tests to this file `tests/sqllogictests/suites/query/functions/02_0074_function_map.test`